en,zh: release note for v1.4.6 (#2422)
pingcap/docs-tidb-operator: en/releases/release-1.4.6.md

---
title: TiDB Operator 1.4.6 Release Notes
---

# TiDB Operator 1.4.6 Release Notes

Release date: July 19, 2023

TiDB Operator version: 1.4.6

> ## Improvements

Reviewer: ```suggestion
## Improvements

- Enable volume resize by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```

## Bug fixes

> - Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))

Reviewer: ```suggestion
- Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
- Fix the issue that graceful drain for TiCDC didn't work if a non-SemVer image tag used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```

---
title: TiDB Operator 1.4.6 Release Notes
---

# TiDB Operator 1.4.6 Release Notes

Release date: July 19, 2023

TiDB Operator version: 1.4.6

## Improvements

> - Enable volume resize by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))

Reviewer 2: ```suggestion
- Enable volume resizing by default ([#5167](https://github.com/pingcap/tidb-operator/pull/5167), [@liubog2008](https://github.com/liubog2008))
```

## Bug fixes

> - Fix the issue that `Error loading shared library libresolv.so.2` when execute backup/restore with BR version >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))

Reviewer 2: ```suggestion
- Fix the issue of `Error loading shared library libresolv.so.2` when executing backup and restore with BR >=v6.6.0 ([#4935](https://github.com/pingcap/tidb-operator/pull/4935), [@Ehco1996](https://github.com/Ehco1996))
```

> - Fix the issue that graceful drain for TiCDC didn't work if a non-SemVer image tag used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))

Reviewer 2: ```suggestion
- Fix the issue that graceful drain for TiCDC does not work if a non-SemVer image tag is used for TiCDC ([#5173](https://github.com/pingcap/tidb-operator/pull/5173), [@csuzhangxc](https://github.com/csuzhangxc))
```
